src/reducers/ReadingList.js

```diff
@@ -0,0 +1,8 @@
+export const readingList = (state = [], action) => {
+  switch (action.type) {
+    case 'SET_READING_LIST':
+      return ( [...state, action.readingList] )
+    default:
```

Comment:
Great work getting this info into `store`!

nitpick: we'll need to rename `readingList` to `book` in the future.